webpack.config.js

```diff
@@ -122,6 +113,9 @@ module.exports = (env, argv) => {
       warningsFilter: /export .* was not found in/
     },
     plugins: [
+      new ESLintPlugin({
+        extensions: ['ts', 'tsx', 'js'],
```

Comment:
```suggestion
        extensions: ['ts', 'tsx', 'js', 'jsx'],
```
in case there are any of those floating around.